pkg/reservations/base.go

```diff
@@ -98,16 +99,18 @@ func (w *TextWriter) WriteAll(out io.Writer, rez []TZReservation) error {
 	return nil
 }
 
-func (w *TextWriter) WriteFilter(out io.Writer, rez []TZReservation, filter Filter) error {
+func (w *TextWriter) WriteFilter(out io.Writer, rez []TZReservation, filter Filter) (error, int) {
```

Comment:
Idiomatically, flip the tuple to be `(int, error)` as `error`, `exists`, `ok`, etc. always seems to be second in the tuple. Great idea to use this, BTW.

Reply:
Sounds good! Made that change in the latest commit.